.circleci/config.yml

```diff
@@ -8,18 +8,50 @@ jobs:
       - checkout
       - run: go get -v -t -d ./...
       - run:
-          name: "Build for Windows, Linux and OSX"
+          name: "Import GPG key"
           command: |
-            FULL_BIN_NAME="terraform-provider-instaclustr_${CIRCLE_TAG}"
-            env GOOS=darwin GOARCH=amd64 make build BIN_NAME="terraform-provider-instaclustr" VERSION=${CIRCLE_TAG}
+            echo ${GPG_PRIVATE_KEY} | base64 --decode > private.key
+            gpg --batch --import --yes private.key
+            rm private.key
+      - run:
+          name: "Configure GPG Agent"
+          command: |
+            echo allow-preset-passphrase > ~/.gnupg/gpg-agent.conf
+            gpg-connect-agent "RELOADAGENT" /bye
+      - run:
+          name: "Preset passphrase"
+          command: |
+            for keygrip in $(gpg --batch --yes --with-colons --with-keygrip --list-secret-keys admin+tfprovider@instaclustr.com | grep grp | sed 's|[grp:,]||g'); do
```

Comment:
[Desired] - could we define the email used here as part of the environment vars too?